### PR TITLE
RHDEVDOCS-5589: Pipelines 1.11.1 release notes

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

enterprise 4.12 and later

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-5589

Link to docs preview:

https://63241--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes#release-notes-1-11-1_op-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-release-notes-1-11.adoc [hunk 1]
```diff
@@ -202,4 +202,40 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+
+* Before this update, when a user who has no write access to a GitHub repository opened a pull request, {pac} CI/CD actions would show as skipped in GitHub. After this update, {pac} CI/CD actions are executed correctly in this case.
+
+* Before this update, {pac} ran CI/CD actions for every pull request into a branch that matched a configured branch name. After this update, {pac} runs CI/CD actions only when the source branch of the pull request matches the exact configured branch name.
+
+* Before this update, a `tkn pac resolve -f` command with a directory specified did not request a GitHub API token even when `{{ git_auth_secret }}` was present in the directory. After this update, the `tkn pac resolve -f` command requests a GitHub API token in this case.
+
+* Before this update, the `tkn pac resolve -f` command failed if a cluster was running an older version of {pipelines-shortname}, which did not yet support the `v1` API. After this update, you can specify the `--v1beta` or `-B` flag in this command dto use the `v1beta` API.
```
Comment (savitaashture) [inline modules/op-release-notes-1-11.adoc:235]: If i remember correctly i think we are not shipping cli in 1.11.1 so if thats the case we need to remove these release notes
cc @concaf, shverma@redhat.com

File: modules/op-release-notes-1-11.adoc [hunk 2]
```diff
@@ -202,4 +202,40 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+
+* After this update, the SHA digest of the `git-init` image corresponds to version 1.10.5, which is the latest released version of the image.
+
+* Before this update, the `tekton-pipelines-controller` leader election used a config map named `config-leader-election`. This name is the default value for knative controllers, so the configuration process for {pipelines-shortname} could affect other controllers and vice versa. After this update, the leader election uses a unique config name, so the configuration process for {pipelines-shortname} does not affect other controllers and is not affected by other controllers.
+
+* Before this update, when a user who has no write access to a GitHub repository opened a pull request, {pac} CI/CD actions would show as skipped in GitHub. After this update, {pac} CI/CD actions are executed correctly in this case.
```
Comment (savitaashture) [inline modules/op-release-notes-1-11.adoc:229]: > After this update, {pac} CI/CD actions are executed correctly in this case.

How about adding what is shows after the update 
I mean before update it use to show `skipped` but after update it shows `Pending approval`

File: modules/op-release-notes-1-11.adoc (hunk 1 above)
Comment (savitaashture) [inline modules/op-release-notes-1-11.adoc:235]: If i remember correctly i think we are not shipping cli in 1.11.1 so if thats the case we need to remove these release notes
cc @concaf, [shverma@redhat.com](mailto:shverma@redhat.com)

File: modules/op-release-notes-1-11.adoc [hunk 3]
```diff
@@ -202,4 +202,40 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+
+* Before this update, you could not set the URL for {tekton-hub} and {artifact-hub} for the hub resolver, so you could use only the preset addresses of {tekton-hub} and Partifact-hub}. After this update, you can configure the URL for {tekton-hub} and {artifact-hub} for the hub resolver, for example, to use a custom {tekton-hub} instance that you installed.
+
+* After this update, the SHA digest of the `git-init` image corresponds to version 1.10.5, which is the latest released version of the image.
+
+* Before this update, the `tekton-pipelines-controller` leader election used a config map named `config-leader-election`. This name is the default value for knative controllers, so the configuration process for {pipelines-shortname} could affect other controllers and vice versa. After this update, the leader election uses a unique config name, so the configuration process for {pipelines-shortname} does not affect other controllers and is not affected by other controllers.
```
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:227]:
```suggestion
* Before this update, the `tekton-pipelines-controller` component used a config map named `config-leader-election`. This name is the default value for knative controllers, so the configuration process for {pipelines-shortname} could affect other controllers and vice versa. With this update, the component uses a unique config name, so the configuration process for {pipelines-shortname} does not affect other controllers and is not affected by other controllers.
```

File: modules/op-release-notes-1-11.adoc (hunk 2 above)
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:229]:
```suggestion
* Before this update, when a user without write access to a GitHub repository opened a pull request, {pac} CI/CD actions would show as `skipped` in GitHub. With this update, {pac} CI/CD actions are shown as `Pending approval` in GitHub.
```

File: modules/op-release-notes-1-11.adoc [hunk 4]
```diff
@@ -202,4 +202,40 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+
+* Before this update, when a user who has no write access to a GitHub repository opened a pull request, {pac} CI/CD actions would show as skipped in GitHub. After this update, {pac} CI/CD actions are executed correctly in this case.
+
+* Before this update, {pac} ran CI/CD actions for every pull request into a branch that matched a configured branch name. After this update, {pac} runs CI/CD actions only when the source branch of the pull request matches the exact configured branch name.
```
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:231]:
```suggestion
* Before this update, {pac} ran CI/CD actions for every pull request into a branch that matched a configured branch name. With this update, {pac} runs CI/CD actions only when the source branch of the pull request matches the exact configured branch name.
```

File: modules/op-release-notes-1-11.adoc [hunk 5]
```diff
@@ -202,4 +202,40 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+
+* Before this update, a `tkn pac resolve -f` command with a directory specified did not request a GitHub API token even when `{{ git_auth_secret }}` was present in the directory. After this update, the `tkn pac resolve -f` command requests a GitHub API token in this case.
```
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:233]:
```suggestion
* Before this update, a `tkn pac resolve -f` command with a directory specified did not request a GitHub API token even when `{{ git_auth_secret }}` was present in the directory. With this update, the `tkn pac resolve -f` command requests a GitHub API token in this case.
```

File: modules/op-release-notes-1-11.adoc (hunk 1 above)
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:235]:
```suggestion
* Before this update, the `tkn pac resolve -f` command failed if a cluster was running an older version of {pipelines-shortname}, which did not yet support the `v1` API. With this update, you can specify the `--v1beta` or `-B` flag in this command to use the `v1beta` API.
```

File: modules/op-release-notes-1-11.adoc [hunk 6]
```diff
@@ -202,4 +202,40 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+
+* Before this update, the `tkn pac resolve -f` command failed if a cluster was running an older version of {pipelines-shortname}, which did not yet support the `v1` API. After this update, you can specify the `--v1beta` or `-B` flag in this command dto use the `v1beta` API.
+
+* Before this update, the `tkn pr logs` command failed to show logs of a pipeline run that uses the cluster resolver, hub resolver, bundle resolver, or git resolver. After this update, the `tkn pr logs` command shows logs of pipeline runs that use any type of resolver.
```
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:237]:
```suggestion
* Before this update, the `tkn pr logs` command failed to show logs of a pipeline run that uses the cluster resolver, hub resolver, bundle resolver, or git resolver. With this update, the `tkn pr logs` command shows logs of pipeline runs that use any type of resolver.
```

File: modules/op-release-notes-1-11.adoc [hunk 7]
```diff
@@ -202,4 +202,40 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+
+* Before this update, in {pipelines-title} 1.11.0, the operator always installed {tekton-chains} and there was no way to disable installation of this component. After this update, in the `TektonConfig` custom resource in the `chains` spec, you can set the `disabled` parameter to `true` to disable installation of {tekton-chains}.
```
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:239]:
```suggestion
* Before this update, in {pipelines-title} 1.11.0, the Operator always installed {tekton-chains} and you would not disable installation of the {tekton-chains} component. With this update, you can set the value of the `disabled` parameter to `true` in the `TektonConfig` CR  to disable installation of {tekton-chains}.
```

File: modules/op-release-notes-1-11.adoc [hunk 8]
```diff
@@ -202,4 +202,40 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+
+* Before this update, if you configured {tekton-chains} on an older version of {pipelines-shortname} using the `TektonChain` custom resource and then upgraded to {pipelines-shortname} version 1.11.0, the configuration information was overwritten. After this update, if you upgrade from an older version of {pipelines-shortname}, {tekton-chains} configuration information is preserved.
```
Comment (shipsing) [inline modules/op-release-notes-1-11.adoc:241]:
```suggestion
* Before this update, if you configured {tekton-chains} on an older version of {pipelines-shortname} using the `TektonChain` CR and then upgraded to {pipelines-shortname} version 1.11.0, the configuration information was overwritten. With this update, if you upgrade from an older version of {pipelines-shortname}, {tekton-chains} configuration information is preserved.
```

**Review by shipsing — CHANGES_REQUESTED** [review]

File: modules/op-release-notes-1-11.adoc (hunk 2 above)
Comment (mramendi) [reply modules/op-release-notes-1-11.adoc]: Fixed, thanks @shipsing

File: modules/op-release-notes-1-11.adoc [hunk 9]
```diff
@@ -202,4 +202,36 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+
+* Before this update, when a user without write access to a GitHub repository opened a pull request, {pac} CI/CD actions would show as `skipped` in GitHub. With this update, {pac} CI/CD actions are shown as `Pending approval` in GitHub.
+
+* Before this update, {pac} ran CI/CD actions for every pull request into a branch that matched a configured branch name. With this update, {pac} runs CI/CD actions only when the source branch of the pull request matches the exact configured branch name.
+
+* Before this update, a `tkn pac resolve -f` command with a directory specified did not request a GitHub API token even when `{{ git_auth_secret }}` was present in the directory. With this update, the `tkn pac resolve -f` command requests a GitHub API token in this case.
```
Comment (savitaashture) [inline modules/op-release-notes-1-11.adoc:233]: @mramendi this one also related to tkn-pac cli 
can we remove it

**Review by ppitonak — APPROVED** [review]

**Review by khrm — COMMENTED** [review]
/lgtm for Pipelines

**Review by concaf — APPROVED** [review]

**Comment by Preeticp** [comment]
Please note: This PR is ready to be reviewed for merging, but we are waiting for the go-live from the team to merge this hence the merge review needed and do not merge labels applied to this PR.

File: modules/op-release-notes-1-11.adoc [hunk 10]
```diff
@@ -202,4 +202,36 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+With this update, {pipelines-title} General Availability (GA) 1.11.1 is available on {product-title} 4.12 and later versions.
+
+[id="fixed-issues-1-11-1_{context}"]
+=== Fixed issues
+
+* Before this update, a task run could fail with a mount path error message, when a running or pending pod was preempted. With this update, a task run does not fail when the cluster causes a pod to be deleted and re-created.
+
+* Before this update, a shell script in a task had to be run as root. With this update, the shell script image has the non-root user ID set so that you can run a task that includes a shell script, such as the `git-clone` task, as a non-root user within the pod.
+
+* Before this update, in {pipelines-title} 1.11.0, when a pipeline run was included in a Tekton bundle, the `kind` value was sometimes wrongly set to `Task`. The issue did not exist in previous versions of {pipelines-title}. With this update, the `kind` value is set correctly.
```
Comment (piyush-garg) [inline modules/op-release-notes-1-11.adoc:219]:
```
when a v1beta1 pipelinerun has pipelineref of bundle, while conversion to v1, the `kind` value was getting wrongly set to `Task`. The issue did not exist in previous versions of {pipelines-title}. With this update, the `kind` value is set correctly.
```

Comment (mramendi) [reply modules/op-release-notes-1-11.adoc:219]: @piyush-garg I don't understand this. In what use cases does a conversion from v1beta1 to v1 happen?

**Comment by bscott-rh** [comment]
> Please note: This PR is ready to be reviewed for merging, but we are waiting for the go-live from the team to merge this hence the merge review needed and do not merge labels applied to this PR.

Please squash down to 1 commit before you are ready to merge. Which versions of the openshift-docs repo should this be cherrypicked to? `main` only, or does it need to be picked to any of the enterprise-4.10+ branches? Thank you

File: modules/op-release-notes-1-11.adoc [hunk 11]
```diff
@@ -202,4 +202,36 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+
+* Before this update, {pac} ran CI/CD actions for every pull request into a branch that matched a configured branch name. With this update, {pac} runs CI/CD actions only when the source branch of the pull request matches the exact configured branch name.
+
+* Before this update, metrics for the {pac} controller were not visible in the {product-title} developer console. With this update, metrics for the {pac} controller are displayed in the developer console.
+
+* Before this update, in {pipelines-title} 1.11.0, the Operator always installed {tekton-chains} and you could not disable installation of the {tekton-chains} component. With this update, you can set the value of the `disabled` parameter to `true` in the `TektonConfig` CR  to disable installation of {tekton-chains}.
+
+* Before this update, if you configured {tekton-chains} on an older version of {pipelines-shortname} using the `TektonChain` CR and then upgraded to {pipelines-shortname} version 1.11.0, the configuration information was overwritten. With this update, if you upgrade from an older version of {pipelines-shortname}, {tekton-chains} configuration information is preserved.
```
Comment (PuneetPunamiya) [inline modules/op-release-notes-1-11.adoc:237]: We need to add one note here 

NOTE:- Data will be preserved only if in previous versions Tekton Chains was installed in the targetNamespace i.e. where TektonConfig was installed (e.g. `openshift-pipelines`)

File: modules/op-release-notes-1-11.adoc [hunk 12]
```diff
@@ -202,4 +202,36 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+With this update, {pipelines-title} General Availability (GA) 1.11.1 is available on {product-title} 4.12 and later versions.
+
+[id="fixed-issues-1-11-1_{context}"]
+=== Fixed issues
+
+* Before this update, a task run could fail with a mount path error message, when a running or pending pod was preempted. With this update, a task run does not fail when the cluster causes a pod to be deleted and re-created.
+
+* Before this update, a shell script in a task had to be run as root. With this update, the shell script image has the non-root user ID set so that you can run a task that includes a shell script, such as the `git-clone` task, as a non-root user within the pod.
+
+* Before this update, in {pipelines-title} 1.11.0, when a pipeline run is defined using {pac}, the definition in the Git repository references the `tekton.dev/v1beta1` API version and includes a `spec.pipelineRef.bundle` entry, the `kind` value was sometimes wrongly set to `Task`. The issue did not exist in previous versions of {pipelines-title}. With this update, the `kind` value is set correctly.
```
Comment (piyush-garg) [inline modules/op-release-notes-1-11.adoc:219]: the `kind` value is getting set everytime, and during creating resources on cluster after conversion to v1

**Comment by PuneetPunamiya** [comment]
LGTM

**Comment by piyush-garg** [comment]
lgtm

**Comment by savitaashture** [comment]
LGTM

**Comment by mramendi** [comment]
This was peer-reviewed by @shipsing but there were changes after that and the point release is out, so we need peer review ASAP (and then I will squash and request merge)

File: modules/op-release-notes-1-11.adoc [hunk 13]
```diff
@@ -202,4 +202,36 @@ With this command, you can also specify a custom GitHub API URL by using the `--
 
 * Before this update, a long pipeline run would be stuck in the running state on the cluster, even after the timeout. This update fixes the issue.
 
-* This update fixes the `tkn pr delete` command for correctly using the `keep` flag. Now, if the value of the `keep` flag equals the number of associated task runs or pipeline runs, the `tkn pr delete` command returns exit code `0` along with a message.
\ No newline at end of file
+* This update fixes the `tkn pr delete` command for correctly using the `keep` flag. Now, if the value of the `keep` flag equals the number of associated task runs or pipeline runs, the `tkn pr delete` command returns exit code `0` along with a message.
```
Comment (tmalove) [inline modules/op-release-notes-1-11.adoc:205]:
```suggestion
* This update fixes the `tkn pr delete` command for correctly using the `keep` flag. With this update, if the value of the `keep` flag equals the number of associated task runs or pipeline runs, the `tkn pr delete` command returns exit code `0` along with a message.
```

File: modules/op-release-notes-1-11.adoc [hunk 14]
```diff
@@ -202,4 +202,36 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+With this update, {pipelines-title} General Availability (GA) 1.11.1 is available on {product-title} 4.12 and later versions.
+
+[id="fixed-issues-1-11-1_{context}"]
+=== Fixed issues
+
+* Before this update, a task run could fail with a mount path error message, when a running or pending pod was preempted. With this update, a task run does not fail when the cluster causes a pod to be deleted and re-created.
+
+* Before this update, a shell script in a task had to be run as root. With this update, the shell script image has the non-root user ID set so that you can run a task that includes a shell script, such as the `git-clone` task, as a non-root user within the pod.
+
+* Before this update, in {pipelines-title} 1.11.0, when a pipeline run is defined using {pac}, the definition in the Git repository references the `tekton.dev/v1beta1` API version and includes a `spec.pipelineRef.bundle` entry, the `kind` parameter for the bundle reference was wrongly set to `Task`. The issue did not exist in previous versions of {pipelines-title}. With this update, the `kind` parameter is set correctly.
```
Comment (tmalove) [inline modules/op-release-notes-1-11.adoc:219]:
```suggestion
* Before this update, in {pipelines-title} 1.11.0, when a pipeline run is defined using {pac}, the definition in the Git repository references the `tekton.dev/v1beta1` API version and includes a `spec.pipelineRef.bundle` entry, the `kind` parameter for the bundle reference was wrongly set to `Task`. The issue did not exist in earlier versions of {pipelines-title}. With this update, the `kind` parameter is set correctly.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#e

File: modules/op-release-notes-1-11.adoc [hunk 15]
```diff
@@ -202,4 +202,36 @@ With this command, you can also specify a custom GitHub API URL by using the `--
...
+
+* Before this update, the `disable-ha` flag was not correctly passed to the `tekton-pipelines` controller, so the High Availability (HA) feature of {pipelines-title} could not be enabled. With this update, the `disable-ha` flag  is correctly passed and you can enable the HA feature as required.
+
+* Before this update, you could not set the URL for {tekton-hub} and {artifact-hub} for the hub resolver, so you could use only the preset addresses of {tekton-hub} and {artifact-hub}. With this update, you can configure the URL for {tekton-hub} and {artifact-hub} for the hub resolver, for example, to use a custom {tekton-hub} instance that you installed.
+
+* With this update, the SHA digest of the `git-init` image corresponds to version 1.10.5, which is the latest released version of the image.
```
Comment (tmalove) [inline modules/op-release-notes-1-11.adoc:225]: Suggestios - Perhaps change 'latest' to 'current'. See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l

**Review by tmalove — APPROVED** [review]

**Comment by mramendi** [comment]
This PR is fully ready for merge review, merging, and cherry-picking. it is time-sensitive because it is release notes for a product update that already got released. Yesterday a merge review was done, but the release itself was pushed out to today and the RNs got a couple of changes since then. Peer review on the final version is done and everything is squashed.

**Comment by mramendi** [comment]
I seem to be unable to remove the do-not-merge label, help needed?

**Comment by bscott-rh** [comment]
Merged to main and cherrypicked to 4.12-4.14.

**Pull request merged**